impeller/geometry/rect.h

```diff
@@ -202,6 +202,9 @@ struct TRect {
 using Rect = TRect<Scalar>;
 using IRect = TRect<int64_t>;
 
+constexpr Rect kLargestCover =
+    Rect::MakeLTRB(-INFINITY, -INFINITY, INFINITY, INFINITY);
```

Comment:
What about adding a `Rect::MakeMaximum` constructor?

Reply:
Handling infinity rects has been a long time coming. I believe infinity coverage shouldn't cause a problem for EntityPass in its current given the frustum/stencil clip cropping. We've already got a few Aiks tests using `DrawPaint` -- including `ColorWheel`. If none of these tests crash when flipping through them then we're probably good to go.

Reply:
Also, `std::numeric_limits<Scalar>::infinity`?

Reply:
Done, I think :)